src/pages/community-champions.js

```diff
@@ -110,16 +123,75 @@ const WhatItMeansToBeAChampionText = ({
     reverse = false,
     title,
     description,
-}) => {
-    return (
-        <WhatItMeansToBeAChampionTextSection reverse={reverse}>
-            <WhatItMeansToBeAChampionRowTitle>
-                {title}
-            </WhatItMeansToBeAChampionRowTitle>
-            <Description collapse>{description}</Description>
-        </WhatItMeansToBeAChampionTextSection>
-    );
-};
+}) => (
+    <WhatItMeansToBeAChampionTextSection reverse={reverse}>
+        <WhatItMeansToBeAChampionRowTitle>
+            {title}
+        </WhatItMeansToBeAChampionRowTitle>
+        <Description collapse>{description}</Description>
+    </WhatItMeansToBeAChampionTextSection>
+);
+
+const LeadByExampleContainer = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.greyDarkThree};
+    border-radius: ${size.xsmall};
+    margin: ${WHAT_IT_MEANS_TO_BE_A_CHAMPION_ROW_GAP} 0 ${size.xlarge} 0;
+    padding: ${LEAD_BY_EXAMPLE_CONTAINER_PADDING};
+    @media ${screenSize.upToLarge} {
+        margin: ${size.large} 0;
+        padding: ${size.mediumLarge};
+    }
+`;
+
+const LeadByExampleItemContainer = styled('div')`
+    display: grid;
+    grid-column-gap: ${size.mediumLarge};
+    grid-template-columns: ${LEAD_BY_EXAMPLE_ICON_WIDTH} auto;
+    @media ${screenSize.upToLarge} {
+        grid-template-columns: ${LEAD_BY_EXAMPLE_ICON_MOBILE_WIDTH} auto;
+    }
+`;
+
+const LeadByExampleSectionTitle = styled(H6)`
+    margin-bottom: ${size.xsmall};
+`;
+
+const LeadByExampleItem = ({ icon, title, description }) => (
+    <LeadByExampleItemContainer>
+        {icon}
+        <div>
+            <LeadByExampleSectionTitle>{title}</LeadByExampleSectionTitle>
+            {description}
+        </div>
+    </LeadByExampleItemContainer>
+);
+
+const CodeOfConductLink = styled(Link)`
+    color: ${({ theme }) => theme.colorMap.lightGreen};
+    text-decoration: none;
+    &:visited {
+        color: ${({ theme }) => theme.colorMap.lightGreen};
+    }
+    &:hover {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+    }
```

Comment:
it's not clear to me why i have to re-define the hover colour here, but it doesn't show up if i don't lol. from my understanding, `CodeOfConductLink` should inherit the hover colour from here - https://github.com/mongodb/devhub/blob/staging/src/components/dev-hub/link.js#L58-L60 ?